test cases/windows/16 gui app/meson.build

```diff
@@ -6,13 +6,15 @@ project('gui_app_test', 'c')
 #
 
 console_lib = static_library('main', 'console_prog.c')
-executable('console', 'dummy.c', link_with: console_lib, gui_app: false)
+executable('console', 'dummy.c', link_with: console_lib, win_subsystem: 'console')
+executable('console2', 'dummy.c', link_with: console_lib, gui_app: false)
```

Comment:
perhaps `tester` should be run on the output executable to verify the subsystem is set correctly.

Reply:
AFAIK it should fail to link if you use WinMain on a console app or plain main on a Windows app.